packages/dynamic_layouts/README.md

```diff
@@ -13,25 +13,138 @@ and the Flutter guide for
 [developing packages and plugins](https://flutter.dev/developing-packages).
```

Comment:
Whomever lands their PR first, the other will likely have merge conflicts here. If so, let me know and I can help resolve if it is a messy merge. :)